json-rpc/docs/type_event.md

```diff
@@ -104,6 +104,46 @@ Event emitted when the url used for off-chain dual attestation checking is rotat
 | new_base_url                | string | New URL endpoint for off-chain communication            |
 | time_rotated_seconds        | u64    | Blockchain time (in seconds) when the rotation occurred |
 
+#### upgrade
+
+TODO: description
```

Comment:
I would put "Event emitted when a WriteSet transaction is committed which causes the state to be updated". Thoughts?